javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,15 @@
+const fullName = 'Cody Miller';
+const isCool = true;
+const totalPets = 1;
+let undefinedVar;
+const undVar2 = undefined;
```

Comment:
This variable name should be `unknown`, and it's value should be `null` rather than `undefined`